src/ResourceManager/Kusto/Kusto.sln

```diff
@@ -0,0 +1,79 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.27004.2005
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Commands.Kusto", "Commands.Kusto\Commands.Kusto.csproj", "{D2A8F744-8DCB-4A13-BA80-EB181FF365AD}"
```

Comment:
Follow the example in the az-cleanup branch: https://github.com/Azure/azure-powershell/blob/az-cleanup/src/ResourceManager/Aks/Aks.sln - this is going to be merged back in early next week.